app/controllers/simplified-account/settings/api-keys/change-name/change-name.controller.js

```diff
@@ -0,0 +1,36 @@
+const { response } = require('@utils/response')
+const formatSimplifiedAccountPathsFor = require('@utils/simplified-account/format/format-simplified-account-paths-for')
+const paths = require('@root/paths')
+const { changeApiKeyName } = require('@services/api-keys.service')
+const DESCRIPTION_VALIDATION = require('@controllers/simplified-account/settings/api-keys/validations')
+const { validationResult } = require('express-validator')
+const formatValidationErrors = require('@utils/simplified-account/format/format-validation-errors')
```

Comment:
I think there is a convention to order these require statements to improve readability, with core modules first and destructured imports last.